tests/phpunit/tests/media.php

```diff
@@ -2194,6 +2194,8 @@ public function test_wp_calculate_image_sizes() {
 	/**
 	 * @ticket 33641
 	 * @requires function imagejpeg
+	 *
+	 * @expectedDeprecated wp_img_tag_add_decoding_attr
```

Comment:
Instead of adding these annotations on all the tests where `wp_img_tag_add_decoding_attr()` is called, we should remove the calls in those tests and replace them in an equivalent way.

For example, in this test here we should be able to remove the call and instead add `add_filter( 'wp_img_tag_add_decoding_attr', '__return_false' );`. The same approach can be taken in a few other tests below which are of similar nature. In other tests, where `wp_img_tag_add_loading_optimization_attrs()` is already called, the `wp_img_tag_add_decoding_attr()` function call can simply be removed.

The only tests that we should continue calling `wp_img_tag_add_decoding_attr()` in are those that explicitly test that, now deprecated function.